deploy.sh

```diff
@@ -20,13 +21,14 @@ push_helm_charts() {
 		fi
 	fi
 
-	mkdir -p tmp/csi-charts/docs/"$PACKAGE"
-	pushd tmp/csi-charts/docs/"$PACKAGE" >/dev/null
+	mkdir -p "$CHARTDIR/csi-charts/docs/$PACKAGE"
+	cp -R ./charts/ceph-csi-"$PACKAGE" "$CHARTDIR/csi-charts/docs/$PACKAGE"
+	pushd "$CHARTDIR/csi-charts/docs/$PACKAGE" >/dev/null
 	helm init --client-only
-	helm package ../../../../charts/ceph-csi-"$PACKAGE"
+	helm package "$CHARTDIR/csi-charts/docs/$PACKAGE/ceph-csi-$PACKAGE"
 	popd >/dev/null
 
-	pushd tmp/csi-charts/docs >/dev/null
+	pushd "$CHARTDIR"/csi-charts/docs >/dev/null
```

Comment:
inconsistent use of `"..."` quoting, only quote `$CHARTDIR` everywhere, or the full path

Reply:
done